### PR TITLE
l10n: Traditional Chinese Support

### PR DESCRIPTION
**Pull request by ZoneTwelve**
Here is the translated content in Traditional Chinese.
If there is additional content that requires translation, please leave a comment.

### CODE REVIEW

File: proxy/src/main/resources/com/velocitypowered/proxy/l10n/messages_zh_TW.properties [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+velocity.error.already-connected-proxy=您已經連線到此代理伺服器了！
+velocity.error.already-connecting=您已經在嘗試連線伺服器了！
+velocity.error.cant-connect=無法連線到 {0}：{1}
+velocity.error.connecting-server-error=無法將您連線至 {0}，請稍後再試。
+velocity.error.connected-server-error=您與 {0} 的連線出現問題。
+velocity.error.internal-server-connection-error=發送內部伺服器連線錯誤。
+velocity.error.logging-in-too-fast=您的登入過於頻繁，請稍後再試。
+velocity.error.online-mode-only=您尚未登入至 Minecraft 帳號。若您已登入，請嘗試重啟您的啟動器。
+velocity.error.player-connection-error=您的連線發生內部錯誤。
+velocity.error.modern-forwarding-needs-new-client=此伺服器僅兼容 Minecraft 1.13 及更高版本。
```
Comment (MagicTeaMC) [inline proxy/src/main/resources/com/velocitypowered/proxy/l10n/messages_zh_TW.properties]: 「相容」is better

Comment (ZoneTwelve) [reply proxy/src/main/resources/com/velocitypowered/proxy/l10n/messages_zh_TW.properties]: Thanks for the correctness.

**Comment by electronicboy** [comment]
Need to figure some stuff out regarding crowdin, afaik, this really should have been submitted there, but there also other stuff we need to look into

**Review by EterNityCH — APPROVED** [review]
LGTM

**Comment by kashike** [comment]
Please note that translations are generally handled on [Crowdin](https://translate.papermc.io/velocity).

**Pull request merged**